Source/player.cpp

```diff
@@ -3152,6 +3156,34 @@ BOOL PM_DoAttack(int pnum)
 		} else if (dObject[dx][dy] > 0) {
 			didhit = PlrHitObj(pnum, dx, dy);
 		}
+#ifdef HELLFIRE
+		if (

```

Comment:
why break the if like this?

Reply:
It's not exactly the easiest to format, but how about this?